include/mxnet/c_api.h

```diff
@@ -1944,8 +1945,8 @@ MXNET_DLL int MXQuantizeSymbol(SymbolHandle sym_handle,
                                const char **excluded_op_names,
                                const uint32_t num_offline, const char **offline_params,
                                const char *quantized_dtype, const bool calib_quantize,
-                               const char *quantize_mode, uint32_t* out_num_calib_names,
-                               const char ***out_calib_names);
+                               const char *quantize_mode, const char *quantize_granularity,
+                               uint32_t* out_num_calib_names, const char ***out_calib_names);
```

Comment:
Break backward compatibility?

Reply:
This API was called by `_quantize_symbol()` only, and  the new argument `quantize_granularity` carries the default vale of `tensor-wise` from Python to C++ perspective. 
For users doing quantization with `quantization.py`, setting `quantize_granularity` to `channel-wise` explicitly to do channel-wise quantization for FullyConnected, otherwise it will do tensor-wise quantization which is same as before (the default behavior).